docs/guides/func_yaml.md

```diff
@@ -62,6 +62,27 @@ volumes:
   path: /workspace/configmap
 ```
 
+### `options`
+Options allows you to set specific configuration for the deployed funciton, ie. it allows you to tweak Knative Service options related to autoscaling and other properties, if these options are not set in the default ones will be used. This options are based on the ones provided by `kn service` [command](https://github.com/knative/client/blob/main/docs/cmd/kn_service_create.md#options).
```

Comment:
```suggestion
Options allows you to set specific configuration for the deployed funciton, allowing you to tweak Knative Service options related to autoscaling and other properties. If these options are not set, the Knative defaults will be used. 
```

Minor wordsmithing, and I think it might be best to avoid referencing the `kn` options, just to avoid confusion among users - but I'm not firm on this suggestion.

Reply:
Oh also a typo... `funciton`.